ovirt-guest-agent-fedora/ovirt-container-hibernate.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/bin/nsenter --mount=/proc/1/ns/mnt -- /usr/share/ovirt-guest-agent/hibernate $@
```

Comment:
ouch I missed this, but the script seems to not be used anywhere.  @vinzenz should we clean this as well?